workflow-controller/controller-service/src/main/java/io/hyscale/controller/util/ManifestAndDeployHelper.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.controller.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import io.hyscale.commons.validator.Validator;
+import io.hyscale.controller.model.EffectiveServiceSpec;
+import io.hyscale.controller.model.WorkflowContext;
+import io.hyscale.controller.validator.impl.ClusterValidator;
+import io.hyscale.controller.validator.impl.DockerValidator;
+import io.hyscale.controller.validator.impl.ManifestValidator;
+import io.hyscale.controller.validator.impl.RegistryValidator;
+import io.hyscale.controller.validator.impl.VolumeValidator;
+
+@Component
+public class ManifestAndDeployHelper {
+    
+    @Autowired
+    private DockerValidator dockerValidator;
+    
+    @Autowired
```

Comment:
Decouple the responsibilities , then it  can be a WorkFlowContextBuilder

Reply:
Ok